src/mist/io/methods.py

```diff
@@ -3804,23 +3827,26 @@ def probe(user, cloud_id, machine_id, host, key_id='', ssh_user=''):
     if not host:
```

Comment:
we should deprecate this function and use the probe_ssh_only and ping functions instead